src/main/java/com/wolox/training/controllers/BookController.java

```diff
@@ -26,22 +25,14 @@
     @Autowired
     private BookRepository bookRepository;
 
-    /**
-     * This method retrieve all books from database and showing them in json format
-     * @return Iterable<Book> object to list all books
-     */
     @GetMapping
     @ApiOperation(value = "Return all books", response = Book.class, responseContainer = "List")
     @ApiResponse(code = 200, message = "Successfully retrieved books")
     public Iterable<Book> getAllBooks(){
         return bookRepository.findAll();
     }
 
-    /**
```

Comment:
it's not necessary to remove the javadocs, in this specific case they don't add too much but in general they can help understand the code 😄